src/main/java/Server/MopeLSPServerLauncher.java

```diff
@@ -42,8 +42,8 @@ public MopeLSPServerLauncher() throws IOException {
 
     /**
      * <p>launches the server and accepts the connection of the client</p>
-     * @throws ExecutionException in case of attempting to retrieve the result of a task that aborted by throwing an exception
-     * @throws InterruptedException in case of a thread gets interrupted
+     * The method launchServer also calls the method get() defined in CompletableFuture class and handles InterruptedException by throwing an
+     * AssertionError and ExecutionException by throwing a RuntimeException
```

Comment:
Again, this is implementation detail. If your implementation is correct, this method will *never* throw an AssertionError, so there is no need to tell the user about this specific detail.